report-service/src/main/java/gov/cdc/dataingestion/kafka/integration/service/KafkaConsumerService.java

```diff
@@ -103,7 +112,7 @@ public void handleMessage(String message,
     @DltHandler
     public void handleDlt(String message, @Header(KafkaHeaders.RECEIVED_TOPIC) String topic) {
         // Once in DLQ -- we can save message in actual db for further analyze
-        log.info("Message: {} handled by dlq topic: {}", message, topic);
+        log.info("Message ID: {} handled by dlq topic: {}", message, topic);
```

Comment:
reflect to comment on line 50. This DLT should be automatically created by this handler; usually topic will be created as for example "elr" then dlt will be created as "elr-dlt". we could add additional logic here to further update the DLT flow. Hence, having dlt in duplicated topic might cause some confusion.